core/site.go

```diff
@@ -82,8 +86,30 @@ func NewSiteFromConfig(
 	site.tariffs = tariffs
 	site.savings = NewSavings(tariffs)
 
+	// maxCurrent is set and > 0: expect load management to be applied
+	if site.MaxCurrent > 0.0 {
+		site.lmEnabled = true
+	} else {
+		// no support for negative values
+		site.lmEnabled = false
+		return nil, errors.New("maxCurrent for the site must not be negative")
```

Comment:
wieso "not negative"?

Reply:
site.lmEnabled = site.MaxCurrent > 0.0

Warum muss das nochmal in einer separaten Variable gespeichert werden?

Reply:
> wieso "not negative"?

Der in der Config angegebene Wert repräsentiert den Wert der Sicherung bzw. maximalen Strom am EVU (Bezug). Aus diesem wird dann später das Budget des zu verteilenden Stroms errechnet. Der tatsächliche aktuelle EVU Strom kann negativ sein (bei Einspeisung).
Hier wird nur der konfigurierte Wert der Sicherung (bzw max erlaubter Bezug am EVU) geprüft.

Reply:
Du vergleichst oben mit > 0 und machst unten Fehlermeldung <0. Was ist mit ==0?

Reply:
fixed. `== 0` deaktiviert LM. `lmEnabled` ist default `false`. Kann das aber explizit aufnehmen. s.u.